packages/block-editor/src/components/block-list/breadcrumb.js

```diff
@@ -17,62 +16,31 @@ import BlockTitle from '../block-title';
  * the root block.
  *
  * @param {string}   props.clientId        Client ID of block.
- * @param {string}   props.rootClientId    Client ID of block's root.
- * @param {Function} props.selectRootBlock Callback to select root block.
+ * @return {WPElement} Block Breadcrumb.
  */
-export class BlockBreadcrumb extends Component {
-	constructor() {
-		super( ...arguments );
-		this.state = {
-			isFocused: false,
+const BlockBreadcrumb = forwardRef( ( { clientId }, ref ) => {
+	const { setKeyboardMode } = useDispatch( 'core/block-editor' );
+	const { rootClientId } = useSelect( ( select ) => {
+		return {
+			rootClientId: select( 'core/block-editor' ).getBlockRootClientId( clientId ),
 		};
-		this.onFocus = this.onFocus.bind( this );
-		this.onBlur = this.onBlur.bind( this );
-	}
-
-	onFocus( event ) {
-		this.setState( {
-			isFocused: true,
-		} );
-
-		// This is used for improved interoperability
-		// with the block's `onFocus` handler which selects the block, thus conflicting
-		// with the intention to select the root block.
-		event.stopPropagation();
-	}
-
-	onBlur() {
-		this.setState( {
-			isFocused: false,
-		} );
-	}
-
-	render() {
-		const { clientId, rootClientId } = this.props;
-
-		return (
-			<div className={ 'editor-block-list__breadcrumb block-editor-block-list__breadcrumb' }>
-				<Toolbar>
-					{ rootClientId && (
-						<>
-							<BlockTitle clientId={ rootClientId } />
-							<span className="editor-block-list__descendant-arrow block-editor-block-list__descendant-arrow" />
-						</>
-					) }
+	} );
+
+	return (
+		<div className={ 'editor-block-list__breadcrumb block-editor-block-list__breadcrumb' }>
```

Comment:
It was there previously, but for convention's sake:

```suggestion
		<div className="editor-block-list__breadcrumb block-editor-block-list__breadcrumb">
```